CHANGES.rst

```diff
@@ -480,6 +480,8 @@ Bug fixes
   - The keyword ``acc`` (for accuracy) is now correctly accepted by
     ``Simplex``. [#3966]
 
+  - Switched first two elements of the returned list of fit_deriv in Voigt1D class. [#4144]
```

Comment:
The changelog entry isn't necessary--the Voigt1D model is new in v1.1 which hasn't been released yet, so any bug fixes to it prior to release don't need to be mentioned here.